### PR TITLE
remix the mix; and default globals for standalone

### PR DESCRIPTION
**Pull request by nvkelso**

### CODE REVIEW

File: themes/bubble-wrap-icons.yaml [hunk 1]
```diff
@@ -1,12 +1,30 @@
...
+    # or import the Refill color theme *after* Bubble Wrap icons
+    sdk_icon_color_style:
+
 styles:
     mapzen_icon_library:
         base: points
         texture: mapzen_icon_library
         # To make Refill's colorize-able icons play nice with other styles
-        mix:             # intentionally blank, is parsed as null (no mix)
+        mix: global.sdk_icon_color_style
```
Comment (tallytalwar) [inline themes/bubble-wrap-icons.yaml]: Hmm one does not need to specify this. The style property merging process will make sure to get the `mix` property from an import.

Comment (tallytalwar) [reply themes/bubble-wrap-icons.yaml]: The same will apply to all other base style (other PRs).

Comment (nvkelso) [reply themes/bubble-wrap-icons.yaml]: Got it. Fixed in https://github.com/tangrams/bubble-wrap/pull/258/commits/36315b87f9f1d6e559aedbe31ced91fbc653810d and all the other PRs have also been updated.

All the example imports in the blog post still work!

Comment (tallytalwar) [reply themes/bubble-wrap-icons.yaml]: Looks good to me.

**Review by tallytalwar — APPROVED** [review]

**Pull request merged**